azure-mgmt-datafactory/azure/mgmt/datafactory/models/expression.py

```diff
@@ -37,4 +37,5 @@ class Expression(Model):
     type = "Expression"
 
     def __init__(self, value):
+        super(Expression, self).__init__()
```

Comment:
Not sure why these changes are here

Reply:
New version of Autorest, this is normal (and was a bug before)